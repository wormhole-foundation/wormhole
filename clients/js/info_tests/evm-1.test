evm address-from-secret 0xcfb12303a19cde580bb4dd771639b0d26bc68353645571a8cff516ab2ee113a0
