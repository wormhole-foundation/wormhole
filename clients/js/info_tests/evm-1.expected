0xbeFA429d57cD18b7F8A4d91A2da9AB4AF05d0FBe
